### PR TITLE
feat(alertStatus): Add Alert Status documentation

### PR DESCRIPTION
**Pull request by mikellykels**
Add new Alert Status documentation for issue alerts.

~~- Still need to add a screenshot~~

[FIXES WOR-1625](https://getsentry.atlassian.net/browse/WOR-1625)

### CODE REVIEW

File: src/docs/product/alerts/alert-types.mdx [hunk 1]
```diff
@@ -30,13 +30,19 @@ In issue alerts, Sentry evaluates the configured alert conditions each time it r
 2. [Filters](/product/alerts/create-alerts/issue-alert-config/#if-conditions-filters) help control noise by triggering an alert only **If** the issue matches the specified criteria.
 3. **Then**, [Actions](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions) specify what should happen when the trigger conditions are met and the filters match.
 
+### Alert Details
+
+The **Alert Details** page shows you the number of times an issue alert rule was triggered over time, grouped in one hour time buckets. Clicking on the Alert Rule name on the "Alert Rules" tab, or the notification you receive will take you to this page. The page also includes details such as the alert rule conditions, the current status of the alert (Warning, Critical, or Resolved), and its details such as when it was created, last modified, and the Team that owns the alert.
```
Comment (imatwawana) [inline src/docs/product/alerts/alert-types.mdx:35]:
```suggestion
The **Alert Details** page shows you the number of times an issue alert rule was triggered over time, grouped in one hour time buckets. Clicking on the alert rule name in the "Alert Rules" tab, or on the notification you receive will take you to this page. The page also includes the alert rule conditions, the current status of the alert (Warning, Critical, or Resolved), and alert details such as when it was created, when it was last modified, and the team that owns the alert.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 2]
```diff
@@ -30,13 +30,19 @@ In issue alerts, Sentry evaluates the configured alert conditions each time it r
...
+
+The **Alert Details** page also includes a list of issues that triggered the alert. You can see what issues caused the alert, and then open that issue's page to find more information.
```
Comment (imatwawana) [inline src/docs/product/alerts/alert-types.mdx:37]:
```suggestion
The **Alert Details** page also includes a list of issues that triggered the alert. You can see which issues caused the alert and then open that issue's page to find more information.
```

**Review by imatwawana — COMMENTED** [review]
Just commenting for now, but I'll circle back and to review again when the screenshot has been added. I've made a few style edits and a couple wording changes to avoid repetition. Otherwise looks good!

**Comment by imatwawana** [comment]
Forgot to add, we'll need to add an EA note on this just after the new heading you've added:

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

File: src/docs/product/alerts/alert-types.mdx (hunk 2 above)
Comment (scttcper) [inline src/docs/product/alerts/alert-types.mdx:37]:
```suggestion
The **Alert Details** page also includes a list of issues that triggered the alert. You can see what issues triggered the alert, and then open that issue's page to find more information.
```

**Review by scttcper — APPROVED** [review]
it looks good to me, but i'd let @imatwawana take another look

File: src/docs/product/alerts/alert-types.mdx [hunk 3]
```diff
@@ -30,13 +30,27 @@ In issue alerts, Sentry evaluates the configured alert conditions each time it r
...
 
+### Alert Details
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you're interested in being an Early Adopter, you can turn your organization's Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+The **Alert Details** page shows you the number of times an issue alert rule was triggered over time, grouped in one hour time buckets. Clicking on the alert rule name in the "Alert Rules" tab, or on the notification you receive will take you to this page. The page also includes the alert rule conditions, the current status of the alert (Warning, Critical, or Resolved), and alert details such as when it was created, when it was last modified, and the team that owns the alert.
```
Comment (imatwawana) [inline src/docs/product/alerts/alert-types.mdx:41]:
```suggestion
The **Alert Details** page shows you the number of times an issue alert rule was triggered over time, grouped in one hour buckets. Clicking on the alert rule name in the "Alert Rules" tab, or on the notification you receive will take you to this page. The page also includes the alert rule conditions, the current status of the alert (Warning, Critical, or Resolved), and alert details such as when it was created, when it was last modified, and the team that owns the alert.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 4]
```diff
@@ -30,13 +30,27 @@ In issue alerts, Sentry evaluates the configured alert conditions each time it r
...
+
+![Alert details illustrates a graph showing the alerts triggered over time.](issue-alert-status-page-details.png)
+
+The **Alert Details** page also includes a list of issues that triggered the alert. You can see which issues triggered the alert and then open that issue's page to find more information.
```
Comment (imatwawana) [inline src/docs/product/alerts/alert-types.mdx:45]:
```suggestion
The **Alert Details** page also includes a list of issues that triggered the alert. You can click on any of the issues in the list to go to that issue's details page for more information.
```

**Review by imatwawana — APPROVED** [review]
A couple more wording fixes, but otherwise looks great!

**Pull request merged**